web/lib/layout/Footer.js

```diff
@@ -13,7 +13,7 @@ const Footer = props => {
       <Box className='footer-content-wrapper twc-px-6 twc-w-full twc-py-[0.75rem] [@media(min-width:1440px)]:twc-max-w-[1440px]'>
         <Box className={'twc-flex twc-flex-wrap twc-items-center twc-justify-between'}>
           <Typography className='twc-mr-2'>
-            {`© 2023 `}
+            {`© 2023-${new Date().getFullYear()} `}
```

Comment:
There is no need to put a year range. We are not using year ranges in our headers, so I don't think we should use it here.

Reply:
Also, as previously explained, the copyright year relates to the year of publishing, not the current year.